Kush/caffe_installation_updated

```diff
@@ -0,0 +1,35 @@
+sudo apt-get install -y --no-install-recommends libboost-all-dev
+sudo apt-get install libprotobuf-dev libleveldb-dev libsnappy-dev libopencv-dev libboost-all-dev libhdf5-serial-dev libhdf5-10 libhdf5-dev libhdf5-cpp-11 libopenblas-dev\
+libgflags-dev libgoogle-glog-dev liblmdb-dev protobuf-compiler
+
+sudo apt install python-pip
+sudo apt install git
+git clone https://github.com/BVLC/caffe
+cd caffe
+cp Makefile.config.example Makefile.config
+
+sudo pip install scikit-image protobuf
+cd python
+for req in $(cat requirements.txt); do sudo pip install $req; done
+
+cd ..
+sudo apt install vim
+sudo vim Makefile.config
+#TODO: Press 'i' on keyboard to insert in the Makefile
+#      CPU-only switch (uncomment to build without GPU support).
+#      Uncomment if you're using OpenCV 3
+#      Also add the following path in INCLUDE_DIRS := /usr/include/hdf5/serial
+#      Press 'ESC' to exit, ':w' and ':x' to save and exit Makefile
+
+find /usr -iname "*hdf5.h*"
+export CPATH="/usr/include/hdf5/serial" 
```

Comment:
Setting CPATH to specific driver folder during compilation can lead to other failures.

To provide a specific path to the gcc build step please use the gcc -I command line option